apps/examples/src/examples/editor-focus/EditorFocusExample.tsx

```diff
@@ -1,34 +1,23 @@
-import { useEffect, useRef, useState } from 'react'
+import { useRef } from 'react'
 import { Editor, Tldraw } from 'tldraw'
 import 'tldraw/tldraw.css'
 
 export default function EditorFocusExample() {
-	const [focused, setFocused] = useState(false)
-	const rEditorRef = useRef<Editor | null>(null)
-
-	useEffect(() => {
-		const editor = rEditorRef.current
-		if (!editor) return
-		editor.updateInstanceState({ isFocused: focused })
-	}, [focused])
+	const editorRef = useRef<Editor | null>(null)
 
 	return (
-		<div
-			style={{ padding: 32 }}
-			onPointerDown={() => {
-				const editor = rEditorRef.current
-				if (editor && editor.getInstanceState().isFocused) {
-					editor.updateInstanceState({ isFocused: false })
-				}
-			}}
```

Comment:
I don't understand why this existed. It makes for some pretty damn confusing behavior in this example IMO.

Reply:
hah yeah agreed 🙃